recipes/dcv_config.rb

```diff
@@ -20,10 +20,10 @@
 
 # Configure the system to enable NICE DCV to have direct access to the Linux server's GPU and enable GPU sharing.
 def allow_gpu_acceleration
-  # On CentOS 7 fix circular dependency multi-user.target -> cloud-init-> isolate multi-user.target.
+  # On CentOS >=7 fix circular dependency multi-user.target -> cloud-init-> isolate multi-user.target.
   # multi-user.target doesn't start until cloud-init run is finished. So isolate multi-user.target
   # is stuck into starting, which keep hanging chef until the 3600s timeout.
-  unless node['platform'] == 'centos' && node['platform_version'].to_i == 7
+  unless node['platform'] == 'centos' && node['platform_version'].to_f >= 7.0
```

Comment:
Wondering if there is a reason we changed from using `to_i` to `to_f`

Reply:
You right, let me restore it.